<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Variable table</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>901</width>
      <height>381</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QPushButton" name="condBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>условие задачи</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>сортировка</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>176</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="sortBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>отсортировать всю таблицу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="keyBtn">
         <property name="text">
          <string>отсортировать по ключу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bdCbx">
         <property name="text">
          <string>выводить только ДР</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="timeBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>замеры времени</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Редактирование</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>170</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="fileBtn">
         <property name="text">
          <string>загрузить из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delCurBtn">
         <property name="text">
          <string>удалить текущую запись</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delBtn">
         <property name="text">
          <string>удалить таблицу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editCurBtn">
         <property name="text">
          <string>редактировать текущую запись</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string>добавить запись</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
